core/server/api/webhooks.js

```diff
@@ -87,6 +130,21 @@ webhooks = {
 
         // Pipeline calls each task passing the result of one to be the arguments for the next
         return pipeline(tasks, options);
+    },
+
+    trigger: function trigger(event, payload, options) {
+        var tasks;
+
+        function doQuery(options) {
+            return models.Webhook.findAllByEvent(event, options);
+        }
+
+        tasks = [
+            doQuery,
+            _.partialRight(makeRequests, payload)
```

Comment:
I couldn't quite tell, does the pipeline utility allow you to add args to a task along with the result of the previous task or is partial application (or something else?) required here?